README.md

```diff
@@ -33,9 +33,13 @@ The system comprises the following components:
 ## Code Prerequisites
 
 In order to build and run the code, you need the following installed:
-  - [Node](https://nodejs.org/) (version `16.13.0`)
-  - [NPM](http://npmjs.org/) (version `8.1.0`)
-  - [Wine](https://www.winehq.org/download), if you would like to generate binaries for Windows.
+  - [Node](https://nodejs.org/en/download/) LTS (version `16.13.0`)
+  - [NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) (version `8.1.0`)
+  - Manager-specic
+    - [Wine](https://www.winehq.org/download), if you would like to generate binaries for Windows.
```

Comment:
Do we really need Wine here?

Reply:
I don't know. Maybe not? But I'd address that in a different PR.

Reply:
👍